modules/developer-preview/pages/collections/rest-api-reference/dropping-a-scope.adoc

```diff
@@ -0,0 +1,61 @@
+= Dropping a Scope
+
+[abstract]
+Scopes can be _dropped_, by means of the REST API.
+
+== Description
+
+Scopes are dropped by means of the DELETE /pools/default/buckets/<bucket_name>/collections/<scope_name>  HTTP method and URI.
```

Comment:
Consider making `<bucket_name>` and `<scope_name>` italic, as in **dropping-a-collection.adoc**.